css/mystyle.css

```diff
@@ -207,6 +206,31 @@ color: #838994;
 	text-align: center;
 
 }
+.clear{
+	clear: both;
+}
+
+@media only screen and (max-width: 1010px) {
+	.assign .illustration > li {
```

Comment:
This makes it so that when the screen gets below 1010px the project pictures stack on top of each other instead of squishing together.